providers/efa/efa.h

```diff
@@ -115,7 +113,6 @@ struct efa_ah {
 
 struct efa_dev {
 	struct verbs_device vdev;
-	uint8_t abi_version;
```

Comment:
Are you sure about this change?

Reply:
This field isn't used anywhere, do you see a reason to keep it?

Reply:
I don't know, maybe for compatibility between providers and libibverbs.